Hexagon.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Lab_1_1
+{
+    class Hexagon: Figure
+    {
+        int x1, y1, x2, y2, x3, y3, x4, y4, x5, y5, x6, y6;
+
+        public Hexagon(int x1, int y1, int x2, int y2, int x3, int y3, int x4, int y4, int x5, int y5, int x6, int y6)
```

Comment:
Also if you work with Points, you should use points in constructor too

Reply:
I didn't understand the comments about the constructor. Do you mean the need to use a collection of points as an input parameter?

Reply:
yep, yo can do this